src/test/java/org/uniqueck/asciidoctorj/exceltableconverter/ExcelTableGeneratorBlockMacroProcessorTest.java

```diff
@@ -40,12 +40,11 @@ void testExcelFileExistAndSheetNameIsProvided() {
     @ParameterizedTest
     @ValueSource(strings = {"simpleExcelTable.xls", "simpleExcelTable.xlsx"})
     void testExcelFileExistAndSheetNameIsProvidedAndSheetExist(String excelFileName) {
-        String actual = convert("excel::" + excelFileName + "[sheetName=Tabelle1]");
+        String actual = convert("excel::" + excelFileName + "[sheetName=Tabelle1,options=\"header\"]");
```

Comment:
If options = header is optional, you can add a test case without options = header